glean-core/tests/ping_maker.rs

```diff
@@ -0,0 +1,120 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+mod common;
+use crate::common::*;
+
+use iso8601;
+
+use glean_core::metrics::*;
+use glean_core::ping::PingMaker;
+use glean_core::{CommonMetricData, Glean, Lifetime};
+
+fn set_up_basic_ping() -> (Glean, PingMaker, PingType, tempfile::TempDir) {
+    let (t, tmpname) = tempdir();
+    let mut glean = Glean::new(&tmpname, GLOBAL_APPLICATION_ID, true).unwrap();
+    let ping_maker = PingMaker::new();
+    let ping_type = PingType::new("store1", true);
+    glean.register_ping_type(&ping_type);
+
+    let metric = BooleanMetric::new(CommonMetricData {
```

Comment:
Should this have a note that we record something to ensure the ping will contain data?